discovery-provider/src/queries/get_health.py

```diff
@@ -147,6 +149,13 @@ def get_health(args, use_redis_cache=True):
         latest_indexed_block_num = db_block_state["number"] or 0
         latest_indexed_block_hash = db_block_state["blockhash"]
 
+    trending_tracks_last_completion = redis.get(
+        trending_tracks_last_completion_redis_key
+    )
+    trending_tracks_age_sec = \
+        (int(time.time()) - int(trending_tracks_last_completion)) \
```

Comment:
@raymondjacobson what happens here if the redis key doesn't exist and we try to cast to an int?

Reply:
i know `int(None)` throws an error

Reply:
The ternary here should protect against that :)

Reply:
you right. idk how i missed that